frontend/pages/_app.js

```diff
@@ -6,13 +6,8 @@ import "highlight.js/styles/github.css";
 import "focus-visible/dist/focus-visible";
 import dynamic from "next/dynamic";
 import { QueryClient, QueryClientProvider } from "react-query";
-
-const HeadSEO = dynamic(() => import("../src/components/HeadSEO"), {
-  ssr: false,
-});
-const HeadLinks = dynamic(() => import("../src/components/HeadLinks"), {
-  ssr: false,
-});
+import HeadLinks from "../src/components/HeadLinks";
+import HeadSEO from "../src/components/HeadSEO";
 const AppContext = dynamic(() => import("../src/AppContext"), {
```

Comment:
Does `AppContext` need to load dynamically?

Reply:
@zomglings Build will fail if you try running it with normal import with following error:
`ReferenceError: window is not defined`

This happens because then it will try to render full HTML of those pages (bodies), while with dynamic it renders only `<head>` and leaves rest as `<script>`

Reply:
Also rendering full HTML does not seem to be making sense for loading speed since when I was trying that - it appears that it will generate BOTH - HTML and same script that will run again after loading, hence chunk size is even bigger and HTML prerendered body indeed can be quite big